src/paths/ForwardingNumber.ts

```diff
@@ -0,0 +1,64 @@
+// This is Generated Source.
+import PathSegment from "../PathSegment";
+import ForwardingNumberInfo from "../definitions/ForwardingNumberInfo";
+import PagingResult from "../PagingResult";
+
+export default class ForwardingNumber extends PathSegment {
+    constructor(prv: PathSegment, id?: string, service?) {
+        super("forwarding-number", id, prv, service);
+    }
+
+    /**
+     *  Add New Forwarding Number
+     */
+    post(body: PostBody): Promise<ForwardingNumberInfo> {
+        return this.getService().send({
+          body: body,
+          method: "post",
+          query: undefined,
+          url: this.getEndpoint(true),
+        }).then(function (res) {
+            return res.json();
+        });
```

Comment:
I know, and this original section from `ForwardingNumber.ts` has to be moved to parent class so that you will not duplicate it every single time in child classes.